searchobject/src/main/java/org/jalau/at18/searchobject/model/objectDetection/ObjectDetectionModel.java

```diff
@@ -19,8 +19,9 @@
 import ai.djl.training.util.ProgressBar;
 import ai.djl.translate.TranslateException;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import org.jalau.at18.searchobject.common.logger.At18Logger;
+//import org.slf4j.Logger;
```

Comment:
Perhaps you could remove the commented lines of code if they are not necessary.

Reply:
It is already removed